pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -57,6 +57,63 @@ type scCmdRet struct {
 var (
 	// Declared as a variable for testing.
 	defaultFS = afero.NewOsFs()
+
+	// Create the config file /etc/suricata/antrea.yaml for Antrea which will be included in the default Suricata config file
+	// /etc/suricata/suricata.yaml.
+	suricataAntreaConfigData = fmt.Sprintf(`%%YAML 1.1
```

Comment:
I understand the concern,
I just added a second eve-log block necessary for L7Visibility, and moved it to var instead of keeping it inside the startSuricata func as in the reconciler_test.go, we were simply copying the whole content and checking if the content is identical which seems redundant.

Please let me know, if we still want to keep it under startSuricata func.

Reply:
Perhaps still add some comments in `startSuricata` to help understand the code?

Reply:
startSuricata func is not part of this PR, let me add comments for the 2 eve-log parts of the code.